templates/hammer_root.yml.epp

```diff
@@ -1,8 +1,8 @@
 <%- |
-  Optional[String] $username,
-  Optional[String] $password,
+  Optional[String]                             $username,
```

Comment:
I'd prefer not to align since it's a lot of work to maintain.
```suggestion
  Optional[String] $username,
```

Reply:
Changed.